### PR TITLE
[v0.1.0] Cleanup docs

### PR DESCRIPTION
**Pull request by indie**

### CODE REVIEW

File: doc/sphinx/source/framework-integration-guides.rst [hunk 1]
```diff
@@ -4,9 +4,61 @@
...
+      (venv)$ pip install -U .
+
+#. See `this file`_ if you want detail about how to run unit tests. To start 
+   working with models, see the `ngraph-neon repo's README`_ to start working
+   with models. 
+
+
 .. _mxnet_intg:
 
 Compile MXNet\* with ``libngraph``
```
Comment (lnguyen-nvn) [inline doc/sphinx/source/framework-integration-guides.rst]: Base on the latest version of ngraph_mxnet (https://github.com/NervanaSystems/ngraph-mxnet/commit/6fad67bf64bab7bc115e76fcd185ff877e2045a2) , we need to fix the  Compile MXNet* with libngraph:


a. Part 4:
USE_NGRAPH option (line -80-) to true with 1 and set the NGRAPH_DIR (line -81-).
Line numbers change to : 
USE_NGRAPH option (line 100) to true with 1 and set the NGRAPH_DIR (line 101)

b. Part 5:
USE_MKL2017 (line -93-) and USE_NNPACK (line -100-).
Line numbers change to : 
USE_MKL2017 (line 113) and USE_NNPACK (line 120).

File: doc/sphinx/source/framework-integration-guides.rst [hunk 2]
```diff
@@ -4,9 +4,61 @@
...
+
+
+
+#. Set the ``NGRAPH_CPP_BUILD_PATH`` and the ``LD_LIBRARY_PATH`` path to the location 
+   where you built the nGraph libraries:
+
+   .. code-block:: bash
+
+      export NGRAPH_CPP_BUILD_PATH=$HOME/ngraph_dist/
+      export LD_LIBRARY_PATH=HOME/ngraph_dist/lib/
```
Comment (lnguyen-nvn) [inline doc/sphinx/source/framework-integration-guides.rst]: Missing $ for parameter HOME in export LD_LIBRARY_PATH=HOME/ngraph_dist/lib/

File: doc/sphinx/source/framework-integration-guides.rst (hunk 1 above)
Comment (indie) [reply doc/sphinx/source/framework-integration-guides.rst]: fixed

File: doc/sphinx/source/framework-integration-guides.rst (hunk 2 above)
Comment (indie) [reply doc/sphinx/source/framework-integration-guides.rst]: Good catch, thanks.  :+1:

**Pull request merged**